packages/iot/src/driver/multispeq/multispeq-protocol-estimator.spec.ts

```
@@ -187,6 +187,40 @@ describe("estimateProtocolDurationMs", () => {
+  it("multiplies a pulseless pre_illumination block by protocol_repeats", () => {
+    expect(
+      estimateProtocolDurationMs([
+        { _protocol_set_: [{ pre_illumination: [2, 200, 10_000], protocol_repeats: 3 }] },
+      ]),
+    ).toBe(30_000);
+  });
+
+  it("sums the multi-LED pre_illumination form", () => {
+    expect(
+      estimateProtocolDurationMs([
+        { _protocol_set_: [{ pre_illumination: [[2, 200, 600_000], [3, 100, 300_000]] }] },
```

ESLint

[error] Replace `·_protocol_set_:·[{·pre_illumination:·[[2,·200,·600_000],·[3,·100,·300_000]]·}]` with `⏎··········_protocol_set_:·[⏎············{⏎··············pre_illumination:·[⏎················[2,·200,·600_000],⏎················[3,·100,·300_000],⏎··············],⏎············},⏎··········],⏎·······`

(prettier/prettier)